app/assets/javascripts/components/enroll/new_account_button.jsx

```diff
@@ -32,9 +39,29 @@ const NewAccountButton = createReactClass({
     // endpoint for the user to register an account on their own.
     if (!course.account_requests_enabled) {
       return (
-        <a data-method="post" href={`/users/auth/mediawiki_signup?origin=${window.location}`} className="button auth signup border margin">
-          <i className="icon icon-wiki-logo" />{I18n.t('application.sign_up_extended')}
-        </a>
+        <>
+          <a
+            data-method="post"
+            href={`/users/auth/mediawiki_signup?origin=${window.location}`}
+            className={`button auth signup border margin ${this.state.disabled ? 'disabled' : ''}`}
+            style={{
+              marginRight: '0'
+            }}
+          >
+            <i className="icon icon-wiki-logo" />
+            {I18n.t('application.sign_up_extended')}
+          </a>
+          {this.state.disabled && (
+          <div className="tooltip-trigger tooltip-small">
+            <img className="info-img" src="/assets/images/info.svg" alt="tooltip default logo" />
+            <div className="tooltip dark large">
+              <p>
+                Your IP address has been blocked from creating an account. Try connecting from a different network or contact your instructor for more information.
```

Comment:
"You cannot currently create an account from this device. This my happen if your IP address is blocked by Wikipedia or has already reached the daily limit for new accounts. Try connecting from a different network."